web5-test-vectors/presentation-exchange/wa-license.json

```diff
@@ -0,0 +1,45 @@
+{
+    "description":"Simple Presentation Exchange",
+    "vectors":[
+       {
+          "description":"presentation exchange for wa drivers license",
+          "input":{
+             "presentationDefinition":{
+                "id":"32f54163-7166-48f1-93d8-ff217bdb0653",
+                "input_descriptors":[
+                   {
+                      "id":"wa_driver_license",
+                      "name":"Washington State Business License",
+                      "purpose":"We can only allow licensed Washington State business representatives into the WA Business Conference",
+                      "constraints":{
+                         "fields":[
+                            {
+                               "path":[
+                                  "$.credentialSubject.dateOfBirth",
+                                  "$.credentialSubject.licenseNumber",
+                                  "$.credentialSubject.licenseState"
+                               ]
+                            }
+                         ]
+                      }
+                   }
+                ]
+             },
+             "credentialJwt":"eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsImtpZCI6ImRpZDprZXk6ejZNa25qSkVXZ0xzaDNSeDM3ZXpUQjJKaE1RY1kxYzN4dnRYS3cxaEF6ZmVlSnpKI3o2TWtuakpFV2dMc2gzUngzN2V6VEIySmhNUWNZMWMzeHZ0WEt3MWhBemZlZUp6SiJ9.eyJpc3MiOiJkaWQ6a2V5Ono2TWtuakpFV2dMc2gzUngzN2V6VEIySmhNUWNZMWMzeHZ0WEt3MWhBemZlZUp6SiIsInN1YiI6ImRpZDprZXk6ejZNa25qSkVXZ0xzaDNSeDM3ZXpUQjJKaE1RY1kxYzN4dnRYS3cxaEF6ZmVlSnpKIiwidmMiOnsiQGNvbnRleHQiOlsiaHR0cHM6Ly93d3cudzMub3JnLzIwMTgvY3JlZGVudGlhbHMvdjEiXSwidHlwZSI6WyJWZXJpZmlhYmxlQ3JlZGVudGlhbCIsIldhRHJpdmVMaWNlbnNlIl0sImlkIjoidXJuOnV1aWQ6OGYzYmU5NjktOTEyYS00MWNjLWFjYjUtZjMzOGQ2MmQ1OGNlIiwiaXNzdWVyIjoiZGlkOmtleTp6Nk1rbmpKRVdnTHNoM1J4MzdlelRCMkpoTVFjWTFjM3h2dFhLdzFoQXpmZWVKekoiLCJpc3N1YW5jZURhdGUiOiIyMDIzLTExLTI5VDIxOjQ5OjIxWiIsImNyZWRlbnRpYWxTdWJqZWN0Ijp7ImlkIjoiZGlkOmtleTp6Nk1rbmpKRVdnTHNoM1J4MzdlelRCMkpoTVFjWTFjM3h2dFhLdzFoQXpmZWVKekoiLCJkYXRlT2ZCaXJ0aCI6IjExLTExLTIwMTEiLCJsaWNlbnNlTnVtYmVyIjoiMTIzNCIsImxpY2Vuc2VTdGF0ZSI6IldBIn19fQ.oGaIuZI9vRcauGI2Zi469mx8KZR1zpNV_HbmFtvWoap6CaPtn-uaG4oxswAfB4ITUs5fePKIgUvdGbBKyWx1CQ"
+          },
+          "output":{
+             "presentationSubmission":{
+                "id":"VPaHaqzslzOqisx02jMS0",
+                "definition_id":"32f54163-7166-48f1-93d8-ff217bdb0653",
+                "descriptor_map":[
+                   {
+                      "id":"wa_driver_license",
+                      "format":"jwt_vc",
+                      "path":"$.verifiableCredential[0]"
+                   }
+                ]
+             }
+          }
+       }
+    ]
+ }
```

Comment:
nit: Indentation here it a little inconsistent. My editor suggested 3-space indentation, but it looks like there's also 4-space indentation as well.

Reply:
hmm I used this, and it lines up with no errors for me:
https://jsonformatter.curiousconcept.com/#

Reply:
The validation is great! Just seems like we have some inconsistent indentation. 

For example, the opening and closing braces for this JSON are in different columns. Maybe that's intentional? Either way, not blocking!